CMakeLists.txt

```diff
@@ -320,15 +320,14 @@ write_basic_package_version_file(
       COMPATIBILITY SameMajorVersion
       )
 
-
```

Comment:
It make sense to make a minimal patch for rebasing reason, please try not to touch indentation/style and other non-functional places.